pkg/test/main_entry.go

```diff
@@ -38,21 +49,63 @@ func MainEntry(m *testing.M) {
 	globalManPath := flag.String(GlobalManPathFlag, "", "path to operator manifest")
 	namespacedManPath := flag.String(NamespacedManPathFlag, "", "path to rbac manifest")
 	singleNamespace = flag.Bool(SingleNamespaceFlag, false, "enable single namespace mode")
+	localOperator := flag.Bool(LocalOperatorFlag, false, "enable if operator is running locally (not in cluster)")
 	flag.Parse()
 	// go test always runs from the test directory; change to project root
 	err := os.Chdir(*projRoot)
 	if err != nil {
 		log.Fatalf("failed to change directory to project root: %v", err)
 	}
-	if err := setup(kubeconfigPath, namespacedManPath); err != nil {
+	if err := setup(kubeconfigPath, namespacedManPath, *localOperator); err != nil {
 		log.Fatalf("failed to set up framework: %v", err)
 	}
+	// setup local operator command, but don't start it yet
+	var localCmd *exec.Cmd
+	var localCmdOutBuf, localCmdErrBuf bytes.Buffer
+	if *localOperator {
+		// taken from commands/operator-sdk/cmd/up/local.go
```

Comment:
I think, this block could be moved to a separate file(like utils) and use it from there.

Reply:
There are a couple of small differences between the cmd/up/local code and this that make a generic function a little bit complicated. For now I think leaving this "as is" is fine and works, but I will make a TODO for that in the code and look into deduplicating this later.